homeassistant/components/plant.py

```diff
@@ -289,8 +301,12 @@ def _load_history_from_db(self):
 
     @property
     def should_poll(self):
-        """No polling needed."""
-        return False
+        """Polling needed to check for outdated measurements.
+
+        The measurements are updated as they arrive from the sensors. But for
+        the maximum age we need to poll the sensor.
+        """
+        return True
```

Comment:
Polling is calling the update() many times a minute, you might be able to a periodic timer if only needing to update the sensor "age".